### PR TITLE
create component Logo

### PR DESCRIPTION
**Pull request by jeremyjorand**

### CODE REVIEW

**Review by mastemm — APPROVED** [review]

**Review by mastemm — DISMISSED** [review]

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import Card from "@components/Cards/Card";
+import Logo from "@components/Logo";
```
Comment (LuigiMenez) [inline frontend/src/App.jsx]: Warning the blank line

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

**Review by mastemm — DISMISSED** [review]
no more blank space

**Review by mastemm — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Ok for me

**Review by LoicBrassart — DISMISSED** [review]
Saw no Evil

**Review by mastemm — APPROVED** [review]

**Comment by LoicBrassart** [comment]
- Fix your conflicts
- Use the "reviewers" *and* the "assignees" fields (otherwise I can't se it)
- Don't assign me at all (Luigi'll do it)

**Review by mastemm — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Ok for me

File: frontend/src/App.jsx [hunk 2]
```diff
@@ -12,6 +15,10 @@ import Autresheros from "./components/Autresheros";
 
 function App() {
   return (
+    <div className="App">
```
Comment (LoicBrassart) [inline frontend/src/App.jsx]: Nope, that'd break your whole App ! :scream:

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Fix this plz (and test)

**Review by Hamza-Luciano — DISMISSED** [review]
modify app.jsx

File: frontend/src/App.jsx [hunk 3]
```diff
@@ -1,28 +1,17 @@
...
 import SGlobal from "./style";
 import Footer from "./components/Footer/Footer";
 import LegalNotice from "./components/LegalNotice/LegalNotice";
-import Contact from "./components/Contact/Contact";
-import Marvel from "./components/Marvel";
-import Starwars from "./components/Starwars";
-import Manga from "./components/Manga";
-import Dccomics from "./components/Dccomics";
-import Autresheros from "./components/Autresheros";
+
```
Comment (LuigiMenez) [inline frontend/src/App.jsx:5]: Blank line

File: frontend/src/App.jsx [hunk 4]
```diff
@@ -1,28 +1,17 @@
...
-        <Route path="/marvel" element={<Marvel />} />
-        <Route path="/starwars" element={<Starwars />} />
-        <Route path="/manga" element={<Manga />} />
-        <Route path="/dccomics" element={<Dccomics />} />
-        <Route path="/autresheros" element={<Autresheros />} />
-        <Route path="/legalnotice" element={<LegalNotice />} />
-        <Route path="/contact" element={<Contact />} />
-      </Routes>
+            <Route path="/legalnotice" element={<LegalNotice />} />
+             </Routes>
```
Comment (LuigiMenez) [inline frontend/src/App.jsx]: Warning indentation

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Verify a prettier for automatic indentation

**Review by Hamza-Luciano — APPROVED** [review]
ok for us :'(

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
Saw no Evil

**Pull request merged**